### PR TITLE
Feature/postings

### LINKED ISSUE #4: homepage/list-of-open-position
As a User, 

I should see the list of all currently open positions with the following information on each card:

- `organization name`
- `date`
- `event name`
- `open positions` number

### LINKED ISSUE #5: homepage/style-open-positions
- Each position is a separate block of information.

- Open positions block should have overflow scroll effect allowing users to scroll inside the element block

### LINKED ISSUE #8: singleEventPage/submit-chosen-position
On the single event page, 

User can only choose and `SUBMIT` one position per event. Upon clicking on the open position, user can see their selected choice and approve it by clicking on `SUBMIT`.

A submitted event shows up on the `upcoming jobs` on the user account

### LINKED ISSUE #12: Display single page view of details of postings
As a user, we want to be able to view a single page view of each of the events that is enabled on click

### PR DESCRIPTION
**Pull request by yiranli624**
#### What's this PR do?
Display open postings on home page(Postings component,  PostingCard component)
Display single event details on newly created page with dynamic url (PostingView component)
Adjust style for user profile side bar
Add style to the home page event postings section
Add style to event details page
Add Route and Link to navigate between pages

#### What does it fix?
n/a
#### Where should the reviewer start?
App/index.js Line 6;
PostingCard/index.js  Line 1;
PostingCard/PostingCard.scss Line 1;
PostingView/index.js Line 1;
PostingView/PostingView.scss Line 1;
Postings/index.js Line 1;
Postings/Postings.scss Line 1;
User/User.scss Line 9;
User/index.js Line 25;
apiCalls.js Line 38

#### How should this be manually tested?
run ```npm start```

#### Screenshots (if appropriate):
Screen Shot 2021-01-06 at 3 51 12 PM

Screen Shot 2021-01-06 at 3 51 30 PM

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -30,7 +30,16 @@ const updateData = (path, action, id) => {
 };
 
 export const apiCalls = {
+
```
Comment (okalimbet) [inline src/apiCalls.js:33]: nice reusable functions for API calls

File: src/Components/PostingView/index.js [hunk 2]
```diff
@@ -1,48 +1,90 @@
...
+            </div>
+            <p>{description}</p>
+            <div className="section-titles">
+              <strong>
+                <p className="posting-info-title">Open Positions</p>
+              </strong>
+              <hr className="section-line" />
+            </div>
+            <div className="posting-position-cards-wrapper">
+              {openJobs.map((job) => (
```
Comment (okalimbet) [inline src/Components/PostingView/index.js:54]: Need to add a hover/active effect to let the user know which job is selected

File: src/Components/PostingView/postingView.scss [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+.posting-left-info-wrapper {
+  width: 70%;
+  margin: 1em;
+}
+
+.posting-right-info-wrapper {
+  width: 30%;
+  margin: 1em;
```
Comment (okalimbet) [inline src/Components/PostingView/postingView.scss:23]: We probably need to add more variables for certain values that we repeat a lot

**Review by okalimbet — APPROVED** [review]
Great Job Us!

File: src/apiCalls.js (hunk 1 above)
Comment (coopterrones) [reply src/apiCalls.js:33]: Great ! will be super helpful moving forward

**Review by coopterrones — COMMENTED** [review]
looks great here !

**Pull request merged**